components/accept-financial-contributions/HostCollectiveCard.js

```diff
@@ -0,0 +1,259 @@
+                <P>{hostCollective.hostFeePercent}%</P>
+              </Flex>
+            </Flex>
+          </Flex>
+        </ModalHeader>
+        <ModalBody>
+          <Fragment>
+            <HTMLContent content={hostCollective.longDescription} />
+            {get(hostCollective, 'settings.tos') && (
+              <Flex flexDirection="column" mx={1} my={4}>
+                <StyledCheckbox
+                  name="tos"
+                  label={
+                    <FormattedMessage
+                      id="createcollective.tos.label"
+                      defaultMessage="I agree with the {hostTosLink} of {name}."
+                      values={{
+                        hostTosLink: (
+                          <ExternalLink href={get(hostCollective, 'settings.tos')} openInNewTab>
+                            <FormattedMessage id="tos" defaultMessage="terms of service" />
+                          </ExternalLink>
+                        ),
+                        name: name,
+                      }}
+                    />
+                  }
+                  required
+                  checked={checked}
+                  onChange={() => setChecked(true)}
+                />
+              </Flex>
+            )}
+            {error && (
+              <MessageBox type="error" withIcon my={[1, 3]}>
+                {error.includes('GraphQL error: ') ? error.replace('GraphQL error: ', 'Error: ') : error}
+              </MessageBox>
+            )}
+          </Fragment>
+        </ModalBody>
+        <ModalFooter>
+          <Flex justifyContent="flex-end" width="100%">
+            <StyledButton
+              buttonType="button"
+              onClick={() => setShow(false)}
+              buttonStyle="secondary"
+              mt={[2, 3]}
+              mb={2}
+              px={3}
+            >
+              {formatMessage(messages.cancel)}
+            </StyledButton>
+            <StyledButton
+              onClick={() => handleApplication()}
+              loading={loading}
+              buttonStyle="dark"
+              mt={[2, 3]}
+              mb={2}
+              ml={3}
+              px={3}
+            >
+              {formatMessage(messages.submit)}
+            </StyledButton>
+          </Flex>
+        </ModalFooter>
+      </Modal>
+    </Fragment>
+  );
+};
+
+HostCollectiveCard.propTypes = {
+  hostCollective: PropTypes.object.isRequired,
+  applyingCollective: PropTypes.object.isRequired,
+  editCollective: PropTypes.func.isRequired,
+};
+
+const applyToHostMutation = gqlV2`
+mutation applyToHost($collective: AccountReferenceInput!, $host: AccountReferenceInput!) {
+  applyToHost(collective: $collective, host: $host) {
+    id
+    slug
+    host {
+      id
+      slug
+    }
+  }
+}
+`;
+
+const addMutation = graphql(applyToHostMutation, {
+  options: {
+    context: API_V2_CONTEXT,
+  },
+  props: ({ mutate }) => ({
+    editCollective: async (collectiveInput, hostInput) => {
```

Comment:
it should be named `applyToHost`, not `editCollective`

Reply:
Good catch - updated